qualification/Jenkinsfile

```diff
@@ -63,30 +63,56 @@ pipeline {
   }
 
   stages {
+    stage('Install requirements to run qualification tests') {
+      steps {
+         sh 'pip install -r qualification/requirements.txt'
+         sh 'pip install --no-deps ".[qualification]" && pip check'
+      }
+    }
 
     /* This stage executes the qualification tests on the latest docker image
      * of katgpucbf that has been previously unit-tested, and generates a report of the
      * qualification test results.
      */
     stage('Run qualification tests') {
        steps {
+         /* Run the actual pytest.
+          * --supprress-tests-failed-exit-code argument does what it says, in
+          * case of failed tests, a nonzero exit code is produced, but that
+          * disrupts the Jenkins pipeline including generation of reports which
```

Comment:
This is a run-on sentence. How about something like:

> Jenkins will not run subsequent steps if pytest returns a non-zero exit code. Passing --suppress-tests-failed-exit-code causes the exit code to be zero if there are failed tests, so that we can still get the report in that case. The junit step will detect the failed tests and mark the build as unstable.